cmd/statreceiver/copy.go

```diff
@@ -8,48 +8,48 @@ import (
 	"log"
 	"time"
 
-	"storj.io/storj/pkg/utils"
+	"github.com/zeebo/errs"
 )
 
 // PacketCopier sends the same packet to multiple destinations
 type PacketCopier struct {
-	d []PacketDest
+	dest []PacketDest
 }
 
 // NewPacketCopier creates a packet copier that sends the same packets to
 // the provided different destinations
-func NewPacketCopier(d ...PacketDest) *PacketCopier {
-	return &PacketCopier{d: d}
+func NewPacketCopier(dest ...PacketDest) *PacketCopier {
+	return &PacketCopier{dest: dest}
 }
 
 // Packet implements the PacketDest interface
 func (p *PacketCopier) Packet(data []byte, ts time.Time) (ferr error) {
-	var errs utils.ErrorGroup
-	for _, d := range p.d {
-		errs.Add(d.Packet(data, ts))
+	var errs errs.Group
```

Comment:
should this variable be named something besides `errs` since `errs` is the name of the zeebo package? I was confused at first when I saw `errs.Add`/`errs.Err` below